tests/test_dataserializer.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of Grid Singularity Exchange.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from typing import Dict
+from gsy_framework.data_serializer import DataSerializer
+
+
+class TestCompressDecompressDict:
```

Comment:
Following our testing guidelines, I'd name this following the name of the class we're testing:
```suggestion
class TestDataSerializer:
```
This helps mapping tests to their class and also helps when using the search feature of IDEs.